packages/sdk/src/Closer.ts

```diff
@@ -0,0 +1,15 @@
+export namespace Closer {
+  export function combine(...closers: Closer[]): Closer {
+    return () => {
+      for (const closer of closers) {
+        try {
+          closer();
+        } catch (e) {
+          // TODO log errors
```

Comment:
We can't log errors here since we won't have a logger. Options are silently ignore or actually not catch. And maybe errors in a closer shouldn't be caught. Or we run all the closers and then throw a combined error.